src/Tribe/Main.php

```diff
@@ -430,7 +430,7 @@ public function should_load_common_admin_css() {
 		$helper = Tribe__Admin__Helpers::instance();
 
 		// Are we on a post type screen?
-		$is_post_type = $helper->is_post_type_screen();
+		$is_post_type = $helper->is_post_type_screen() || 'post' === get_current_screen()->base;
```

Comment:
This feels like it will include a lot of other scripts in all pages. Can we track down which particular Assets we are trying to insert? I would like to see the change happening closer to the actual asset we need to include.

Reply:
I've identified the 2 files that we need to load and modified the approach slightly.